README.md

```diff
@@ -0,0 +1,103 @@
+## Generate a new Maven project ##
+
+- Run below command from Command line tool
+
+```text
+mvn archetype:generate -DartifactId=selenium-learning -DgroupId=com.mytest -DarchetypeGroupId=org.apache.maven.archetypes -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4
+```
+
+- Run below command to download Maven Dependencies, compile, and build project
+
+```text
+mvn clean package -e
+```
+OR,
+
+```text
+mvn clean install -e
+```
+**NOTE:** Use argument `-e` to console error messages while building the project 
+
+- Run below command to run unit tests against compile source code using available testing framework
+
+```text
+mvn test
+```
+
+## Add required dependency for Maven project ##
+
+- Update/Modify the deppendencies in `pom.xml`. Refer [Maven Central Repository](https://central.sonatype.com/) to know about the latest version of dependencies.
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/selenium-automation/pull/2#issuecomment-2155391508>#6f5971</a><span> - 06/08/2024, 12:54 am</span>
<br>

<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Typographical error in the word "deppendencies". It should be "dependencies".
</div>
<div id="fix">
<b>Fix:</b> Correct the spelling of "deppendencies" to "dependencies".
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div>
<pre><code>
 @@ -29,7 +29,7 @@
  - Update/Modify the deppendencies in 'pom.xml'. Refer [Maven Central Repository](https://central.sonatype.com/) to know about the latest version of dependencies.
 + - Update/Modify the dependencies in 'pom.xml'. Refer [Maven Central Repository](https://central.sonatype.com/) to know about the latest version of dependencies.
</code></pre>
</div>
</details>
</div>
<br>


</div>